src/application/clients/crux-explorer-client.ts

```diff
@@ -0,0 +1,22 @@
+import { setCacheStorage } from "../..";
+import { CruxSpec } from "../../core/entities/crux-spec";
+import { CruxUser } from "../../core/entities/crux-user";
+import { ICruxBlockstackInfrastructure } from "../../core/interfaces";
+import { ICruxUserRepository } from "../../core/interfaces/crux-user-repository";
+import { BlockstackCruxUserRepository } from "../../infrastructure/implementations/blockstack-crux-user-repository";
+import { inmemStorage } from "../../packages";
+import { CruxId } from "../../packages/identity-utils";
+
+export class CruxExplorerClient {
+    private cruxBlockstackInfrastructure: ICruxBlockstackInfrastructure;
+    private cruxUserRepository: ICruxUserRepository;
+    constructor() {
+        setCacheStorage(new inmemStorage.InMemStorage());
```

Comment:
If cache changes are done, you can implement the same for CruxExplorerClient

Reply:
Updated with new cache changes